test/components/plots/plotTests.ts

```diff
@@ -155,9 +155,12 @@ describe("Plots", () => {
       var svg = TestMethods.generateSVG(400, 400);
       var plot = new Plottable.Plot();
 
+      var dataset1 = new Plottable.Dataset([{value: 0}, {value: 1}, {value: 2}]);
+      var dataset2 = new Plottable.Dataset([{value: 1}, {value: 2}, {value: 3}]);
+
       // Create mock drawers with already drawn items
       // HACKHACK #1984: Dataset keys are being removed, so this is the internal key
-      var mockDrawer1 = new Plottable.Drawers.AbstractDrawer("_0");
+      var mockDrawer1 = new Plottable.Drawers.AbstractDrawer(dataset1);
```

Comment:
Remove the `HACKHACK` comment since you're fixing the problem. You actually get to close #1984 in fact.